openshift/template.yaml

```diff
@@ -11,18 +11,18 @@ objects:
   kind: DeploymentConfig
   metadata:
     labels:
-      service: f8a-worker-scaler
-    name: f8a-worker-scaler
+      service: f8a-worker-scaler-${DC_NAME}
+    name: f8a-worker-scaler-${DC_NAME}
```

Comment:
I *think* there is a limit on how long the name can be. My guess would be that this won't fit.